docs/FAQ.md

```diff
@@ -37,6 +37,23 @@ See [revsets] and [templates] for further guidance.
 
 Use `jj log -r ..`. The `..` [operator] lists all visible commits in the repo, excluding the root (which is never interesting and is shared by all repos).
 
+### Can I monitor how `jj log` evolves?
+
+The simplest way to monitor how the history as shown by `jj log` evolves is by using the `watch(1)` command (or [hwatch](https://github.com/blacknon/hwatch) or [viddy](https://github.com/sachaos/viddy)).
+For example:
+
+```sh
+watch jj --ignore-working-copy log --color=always
```

Comment:
Important: You need `--color`. I'd make this:

     watch --color -- jj --ignore-working-copy --color=always log